src/main/scala/scala/concurrent/stm/japi/STM.scala

```diff
@@ -53,7 +55,7 @@ object STM {
    * Create an empty TMap. Return a `java.util.Map` view of this TMap.
    * @return a new, empty `TMap.View` wrapped as a `java.util.Map`.
    */
-  def newMap[A, B](): JMap[A, B] = JavaConversions.mutableMapAsJavaMap(newTMap[A, B])
```

Comment:
These should probably be changed to use `JavaConverters` instead; not sure what the equivalent code would be, so no changes here for now.